charts/jaeger/templates/cleanup-es-indexes-cronjob.yaml

```
@@ -28,8 +28,12 @@ spec:
             {{- end }}
         spec:
           serviceAccountName: {{ template "jaeger.esIndexCleaner.serviceAccountName" . }}
+          securityContext:
+{{ toYaml .Values.esIndexCleaner.podSecurityContext | indent 12 }}
           containers:
           - name: {{ include "jaeger.fullname" . }}-es-cleanup
```

```suggestion
          - name: {{ include "jaeger.fullname" . }}-es-index-cleaner
```

nit: just to match the component name

---

Could you please adjust for all instanced of `{{ include "jaeger.fullname" . }}-es-cleanup` ? 🙏